web/Dockerfile

```diff
@@ -39,18 +39,18 @@ ARG REDIRECT_URI=http://localhost/
 ARG AUTH_SCOPE=""
 ARG CLIENT_ID=""
 ARG TENANT_ID=""
-ENV REACT_APP_AUTH_SCOPE=$AUTH_SCOPE
-ENV REACT_APP_AUTH=$AUTH_ENABLED
-ENV REACT_APP_AUTH_CLIENT_ID=$CLIENT_ID
-ENV REACT_APP_AUTH_TENANT=$TENANT_ID
-ENV REACT_APP_TOKEN_ENDPOINT=https://login.microsoftonline.com/${REACT_APP_AUTH_TENANT}/oauth2/v2.0/token
-ENV REACT_APP_AUTH_ENDPOINT=https://login.microsoftonline.com/${REACT_APP_AUTH_TENANT}/oauth2/v2.0/authorize
-ENV REACT_APP_LOGOUT_ENDPOINT=https://login.microsoftonline.com/${REACT_APP_AUTH_TENANT}/oauth2/logout
-ENV REACT_APP_AUTH_REDIRECT_URI=$REDIRECT_URI
+ENV VITE_AUTH_SCOPE=$AUTH_SCOPE
+ENV VITE_AUTH=$AUTH_ENABLED
+ENV VITE_AUTH_CLIENT_ID=$CLIENT_ID
+ENV VITE_AUTH_TENANT=$TENANT_ID
+ENV VITE_TOKEN_ENDPOINT=https://login.microsoftonline.com/${VITE_AUTH_TENANT}/oauth2/v2.0/token
+ENV VITE_AUTH_ENDPOINT=https://login.microsoftonline.com/${VITE_AUTH_TENANT}/oauth2/v2.0/authorize
+ENV VITE_LOGOUT_ENDPOINT=https://login.microsoftonline.com/${VITE_AUTH_TENANT}/oauth2/logout
+ENV VITE_AUTH_REDIRECT_URI=$REDIRECT_URI
 
 WORKDIR /code
 COPY ./ ./
-RUN yarn install  --immutable --immutable-cache
+RUN yarn install --immutable

```

Comment:
Probably shouldn't make this change, but I couldn't make `docker compose up --build` work without it - it just complained about the packages not being cached. I might have done something wrong with yarn pnp here, please advise.

Reply:
As far as I can tell, you don't need neither flags: https://yarnpkg.com/cli/install#examples in this case, since we don't want validation inside our dockerfile.

However, I think the issue can be solved by adding
```
supportedArchitectures:
  os:
    - 'linux-arm64'
```
to the .yarnrc.yml file

I agree that we have tested pnp and it is not good. There is always something wrong with it. Yes, we do save some time on installing dependencies both locally and in CI, and we avoid some "it works on my machine" because we've forgotten to update our packages. But we also get a lot of weird bugs. I think it's time to stop using pnp, because it just isn't worth it